### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by RQ16**

### CODE REVIEW

File: views/layouts/main.hbs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" type="text/css" href="./style/style.css">
+    <meta charset="utf-8">
+    <title>Main hbs</title>
```
Comment (jwld) [inline views/layouts/main.hbs]: maybe change the title to NazBay for now

File: views/layouts/main.hbs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" type="text/css" href="./style/style.css">
```
Comment (jwld) [inline views/layouts/main.hbs]: might be good to add a meta viewport tag - see here https://www.w3schools.com/css/css_rwd_viewport.asp

Comment (jwld) [reply views/layouts/main.hbs]: makes it scale nicely on mobile

**Review by jbesraa — APPROVED** [review]
👍🏼

**Pull request merged**